Please provide build clean

I set a wrong path for cmake. It is error out. However it does not recover after I set correct path. I think we need build.sh clean

izhih@xsjminm50:/proj/xsjhdstaff4/lizhih/git/XRT/build$ ./build.sh 
-- Checking for module 'libdrm'
--   --short-errors: unknown option
CMake Error at /usr/share/cmake-3.5/Modules/FindPkgConfig.cmake:367 (message):
  A required package was not found
Call Stack (most recent call first):
  /usr/share/cmake-3.5/Modules/FindPkgConfig.cmake:532 (_pkg_check_modules_internal)
  CMakeLists.txt:14 (pkg_check_modules)

--- a/build/build.sh
+++ b/build/build.sh
@@ -1,11 +1,49 @@
 #!/bin/bash
 
+BUILDDIR=$(readlink -f $(dirname ${BASH_SOURCE[0]}))
+
+usage()
+{
+    echo "Usage: build.sh [options]"
+    echo
+    echo "[-help]                    List this help"
+    echo "[clean|-clean]             Remove build directories"
+
+    exit 1
+}
+
+clean=0
+while [ $# -gt 0 ]; do
+    case "$1" in
+        -help)
+            usage
+            ;;
+        clean|-clean)
+            clean=1
+            shift
+            ;;
+        *)
+            echo "unknown option"
+            usage
+            ;;
+    esac
+done
+
 here=$PWD
+cd $BUILDDIR
+
+if [ $clean == 1 ]; then
+    echo $PWD
+    echo "/bin/rm -rf Release Debug"
+    /bin/rm -rf Release Debug
+    exit 0
+fi
+
 mkdir -p Debug Release
 cd Debug
 cmake -DCMAKE_BUILD_TYPE=Debug ../../src
 make -j4 DESTDIR=$PWD install
-cd $here
+cd $BUILDDIR
 
 cd Release
 cmake -DCMAKE_BUILD_TYPE=Release ../../src

